.travis.yml

```diff
@@ -3,12 +3,13 @@ language: php
 php:
   - 7.1
   - 7.2
+  - 7.3
 
 dist: trusty
 sudo: required
 
 before_script:
-  - phpenv config-rm xdebug.ini
+  - if [ $TRAVIS_PHP_VERSION != "7.3" ]; then phpenv config-rm xdebug.ini; fi
```

Comment:
@jgrossi If you have a better solution, please, update this patch. Otherwise, should be RTM :blush:

Reply:
actually, I added the xdebug back to enable code coverage. I changed some things but it seems to be working fine. thanks for sharing @carusogabriel 💪🔥